fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraTombstones.java

```diff
@@ -107,7 +113,20 @@ public Response put() {
 
     @OPTIONS
     public Response options() {
-        return Response.ok().header(ALLOW, "DELETE").build();
+        final var resource = resource();
+        if (!(resource instanceof Tombstone)) {
+            // If the resource is not deleted there is no tombstone.
+            return Response.status(Response.Status.NOT_FOUND).build();
+        }
+
+        final var response = Response.ok().allow();
+        final Tombstone tombstone = (Tombstone) resource;
+        final var deletedResource = tombstone.getDeletedObject();
+        if (deletedResource.getArchivalGroupId().isEmpty()) {
+            response.allow("DELETE");
+        }
+
+        return response.build();
```

Comment:
This works for OPTIONS requests, though for Archival Group members you get a blank header
```
Allow:
```
But we still return 
```
Allow: DELETE
```
for all the other methods that return a `406 Method Not Allowed`. This is built in the `methodNotAllowed()` on lines 147-149.